lib/resolveurl/plugins/streamwire.py

```diff
@@ -0,0 +1,34 @@
+"""
+    Plugin for ResolveUrl
+    Copyright (C) 2020
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from __resolve_generic__ import ResolveGeneric
+from lib import helpers
+
+
+class StreamWireResolver(ResolveGeneric):
+    name = "streamwire"
+    domains = ['streamwire.net']
+    pattern = r'(?://|\.)(streamwire\.net)/(?:embed-)?([0-9a-zA-Z]+)'
+
+    def get_media_url(self, host, media_id):
+        return helpers.get_media_url(self.get_url(host, media_id),
+                                     patterns=[r'''file:\s*"(?P<url>[^"]+)'''],

```

Comment:
Are you sure about the pattern?
In your earlier PR you had `src:` instead of `file:`

Reply:
I am not sure, maybe i am wrong with file and scr is right

Reply:
Can You help me please?

Reply:
Yes, the patterns should be like this for this hoster
```patterns=[r'''src:\s*"(?P<url>[^"]+)''']```
Please update the PR with this change

Reply:
i am doing a new PR, Thanks